x-pack/plugins/apm/ftr_e2e/cypress/integration/read_only_user/service_overview/header_filters.spec.ts

```diff
@@ -46,11 +46,6 @@ const apisToIntercept = [
       '/internal/apm/services/opbeans-node/service_overview_instances/main_statistics?*',
     name: 'instancesMainStatisticsRequest',
   },
-  {
```

Comment:
Does this fail because of https://github.com/elastic/kibana/issues/127704?

Reply:
Well, this is a new bug opened, but this was actually something we removed intentionally https://github.com/elastic/kibana/pull/124933